### PR TITLE
BREAKING CHANGE: re-license project under AGPLv3

### LINKED ISSUE #494: Choose another license
In order to make sure that no one tries to monetize this platform , we will have to find another license.
Maybe https://github.com/OWASP/owasp-mastg/blob/master/License.md ? 

This can help people to feel better protected in their spare time efforts they put in this project.

### PR DESCRIPTION
**Pull request by bendehaan**
Re-license the project to AGPLv3 to guard against use in closed-source projects. Closes #494 

What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [ ] Additional documentation
-   [x] Something else

Checklist:

-   [x] All the contributions made are solely the work of me and my co-authors
-   [ ] I tested the changes in this PR (if applicable)
-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [x] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,661 @@
...
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+                    GNU AFFERO GENERAL PUBLIC LICENSE
+                       Version 3, 19 November 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```
Comment (commjoen) [inline LICENSE:4]: Can you please change this to a similar header as we have with MIT?

**Review by commjoen — CHANGES_REQUESTED** [review]
Awesome! Let’s make sure we have the right license holders defined.

File: LICENSE (hunk 1 above)
Comment (bendehaan) [reply LICENSE:4]: No, this license file should not be modified. Added a line to the README.md instead

Comment (commjoen) [reply LICENSE:4]: ahhh thanks! I guess we need to do the same for https://owasp.org/www-project-wrongsecrets/#:~:text=secret%20detection%20testbed.-,Licensing,-This%20program%20is right ?

Comment (commjoen) [reply LICENSE:4]: Can you check https://github.com/OWASP/www-project-wrongsecrets/pull/8 please? so we can merge them both in :). and then maybe have a blog on why we changed licenses :) ?

**Review by commjoen — APPROVED** [review]
thank you!

**Pull request merged**